src/Symfony/Component/Security/Guard/Firewall/GuardAuthenticationListener.php

```diff
@@ -86,7 +88,7 @@ public function handle(GetResponseEvent $event)
         }
     }
 
-    private function executeGuardAuthenticator($uniqueGuardKey, GuardAuthenticatorInterface $guardAuthenticator, GetResponseEvent $event)
+    private function executeGuardAuthenticator($uniqueGuardKey, GuardAuthenticatorLegacyInterface $guardAuthenticator, GetResponseEvent $event)
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [The method executeGuardAuthenticator() has a Cyclomatic Complexity of 14. The configured cyclomatic complexity threshold is 10.](https://www.codacy.com/app/git_9/symfony/file/2331610611/issues/source?bid=3207821&fileBranchId=3207918#l91)